setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
-name = pricecypher_api
-version = 0.0.1
+name = pricecypher-sdk
```

Comment:
I see you changed pricecypher_python_api to pricecypher_python_sdk. In setup.cfg however the URL and buck tracker URL still point to pricecypher_python_api.

Reply:
Good one, updated